### DISCUSSION TITLE
Ability to customize pagination link `onEachSide`

### DISCUSSION THREAD

**Discussion by misbahansori** [filamentphp/filament#13772] [Ideas]
Hi,

I'd like to request a feature to customize the `onEachSide` value in Laravel's pagination. Currently, it seems there's no way to adjust this value. It would be fantastic if we could have the ability to set `onEachSide` accordingly.

![Screenshot 2024-08-01 at 15 53 33](https://github.com/user-attachments/assets/70c51d1c-9035-4d89-bf5e-7e56956fda7d)

Thank you!

**Comment by indigoram89** [comment]
+1

**Comment by avinmaster** [comment]
+1

**Comment by WillieOng-HK** [comment]
```
use Filament\Resources\Pages\ListRecords;
use Illuminate\Contracts\Pagination\Paginator;
use Illuminate\Database\Eloquent\Builder;

class ListXXX extends ListRecords
{
  protected function paginateTableQuery(Builder $query): Paginator
  {
      return $query->paginate()->onEachSide(2);
  }
}
```

![image](https://github.com/user-attachments/assets/4e97c57c-7d52-4c93-abad-535da72d50fc)

**Reply by indigoram89** [reply]
yes, but it is not a global setup for all tables
